### PR TITLE
Add greatings

### PR DESCRIPTION
**Pull request by TesteThalya**

### CODE REVIEW

**Review by thalya-coimbra — DISMISSED** [review]
Nice greatings

**Review by thalya-coimbra — APPROVED** [review]

**Comment by thalya-codes** [comment]
Hello there. Are you should be add one more example of code in the READ.ME, please?

File: README.md [hunk 1]
```diff
@@ -23,4 +23,6 @@ read me
 [] Item 11
 
 ## Title 2
-* Olá | Hello | Konnichiwa | 
\ No newline at end of file
+* Olá | Hello | Konnichiwa | 
+
+``console.log(1+3)``
```
Comment (thalya-coimbra) [inline README.md]: Hi there, nice job! Could you be add one more example of code in the READ.ME, please?

**Review by thalya-coimbra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TesteThalya) [reply README.md]: Sure. I'll add

Comment (TesteThalya) [reply README.md]: Done

**Review by thalya-coimbra — APPROVED** [review]

**Pull request merged**